server/server.go

```diff
@@ -945,5 +956,11 @@ func (s *Server) reloadConfigFromKV() error {
 		s.storage.SwitchToDefaultStorage()
 		log.Info("server disable region storage")
 	}
+
+	if s.cfg.EnableConfigManager {
+		err = s.cfgManager.Reload(s.storage)
```

Comment:
Does it happen before starting grpc server?

Reply:
It happened after starting the gRPC server

Reply:
Can we ensure it must be done before any `config.Create/Get/Update/Delete`?

Reply:
Could you add some comments about the order of `Reload` and serving requests?